pkg/kubelet/dockershim/network/hostport/hostport_syncer.go

```diff
@@ -137,6 +137,11 @@ func writeLine(buf *bytes.Buffer, words ...string) {
 	buf.WriteString(strings.Join(words, " ") + "\n")
 }
 
+func writeBytes(buf *bytes.Buffer, bytes []byte) {
```

Comment:
if possible move this to pkg/util so that code is not repeated

Reply:
Name of function is not clear it writes bytes plus adds new line

Reply:
Renamed to writeBytesLine (for better consistency with writeLine above).

Re moving to pkg/util - I'm not a fan of large pkg/util, because when you need something you will not find it anyway in such case. This function is so extremely simple that because of lack of obvious place to put it, I'm leaving it here.